cpp/Platform.Converters/CachingConverterDecorator.h

```diff
@@ -1,15 +1,28 @@
-﻿namespace Platform::Converters
+﻿#pragma once
+#include <unordered_map>
+#include <memory>
+
+#include "IConverter[TSource, TTarget].h"
+
+namespace Platform::Converters
 {
     template <typename ...> class CachingConverterDecorator;
     template <typename TSource, typename TTarget> class CachingConverterDecorator<TSource, TTarget> : public IConverter<TSource, TTarget>
```

Comment:
`IConverter<TSource, TTarget>` interface does not exist in C++ version of the code, so there is no need to inherit from it.